pkg/services/featuremgmt/toggles_gen.csv

```diff
@@ -177,6 +177,8 @@ ssoSettingsLDAP,experimental,@grafana/identity-access-team,false,false,false
 failWrongDSUID,experimental,@grafana/plugins-platform-backend,false,false,false
 zanzana,experimental,@grafana/identity-access-team,false,false,false
 passScopeToDashboardApi,experimental,@grafana/dashboards-squad,false,false,false
+passTimeRangeToDashboardApi,experimental,@grafana/dashboards-squad,false,false,false
+passFiltersToDashboardApi,experimental,@grafana/dashboards-squad,false,false,false
```

Comment:
Why do we need two toggles for the reload?

Reply:
Thought more granular control can help with future development.

Reply:
Unified under one feature toggle